### ISSUE TITLE
Add option to process specific integrations immediately rather than in the queue

### ISSUE THREAD

**Issue by Mosnar** [verbb/formie#1152]
### What are you trying to do?

I'm using the element integration to power a user registration form. I think it would be more ideal if this integration ran immediately. This would solve two issues:
- Element validation errors could be accounted for during the form submission validation. This prevents errors related to required user fields that aren't present in the registration form.
- Verification emails and critical event listeners avoid being trapped behind a potentially full or stalled queue worker. If I register, I expect to get a verification email immediately, not in 15 minutes when Craft finishes pruning revisions.

Related: #1151

### What's your proposed solution?

Add a generic option to integration settings to "Process Immediately", which would direct Formie to execute the integration immediately rather than pushing it to the back of the line.

**Comment by engram-design** [comment]
Yeah this is a pretty valid use-case I suppose, and while not ideal, not really any way around it other than processing it outside of the queue (or proving the option to do so).

As for validation, that's tricky, because integrations (and email notifications) are done after a submission is successful, so it would need to back-track a successful submission to make it invalid is an error was encountered in the integration.

**Comment by a-am** [comment]
I have also run into this issue.  When creating registration a user is unable to login directly after registration. They can't login until the queue is run.

**Comment by jishi** [comment]
I have run into a similar problem here. I'm using a form to create Entries (pages) and the problem is two-fold:

Entry might not exist when you are at the success page of the form submit.
There is no way to actually link to the newly created Entry form a success page, nor in an email notification, which might be preferable. 

The second problem might be solvable by allowing email notifications to be triggered when all integrations (or at least one) are finished, but maybe that is not possible the way the queue system works.

**Comment by engram-design** [comment]
Added in [4.0.0-beta.6](https://github.com/verbb/formie/releases/tag/4.0.0-beta.6). Integrations now have a whole dedicated section on running in a particular order, which items run in a queue, before/after notifications, etc.
